### ISSUE TITLE
frontier.svg is using non-standard svg attribute, causing build errors

### ISSUE THREAD

**Issue by Yuripetusko** [lifinance/widget#75]
Many build tools are transforming svg attributes to camelCase for jsx to use. frontier.svg is using an uncommon attribute `xmlns:xodm` which is causing an issue when this svg is being imported 

```
error - ../../node_modules/@lifi/wallet-management/walletIcons/frontier.svg
SyntaxError: unknown: Namespace tags are not supported by default. React's JSX doesn't support namespace tags. You can set `throwIfNamespace: false` to bypass this warning.
```

is it possible to remove this attribute?

**Comment by chybisov** [comment]
@Yuripetusko thanks for reporting the issue 🙏 Should be fixed in `v2.0.0-beta.3` 🙂

**Comment by Yuripetusko** [comment]
Thank you
